recipes/lunardate/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "lunardate" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6c9c96d9f01522a10ab35df1a9b48707ae64a086f13fd34498b43f465918cc6f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - lunardate
+
+about:
+  home: https://github.com/lidaobing/python-lunardate
+  license: GPL-3.0
+  license_family: GPL
+  summary: 'A Chinese Calendar Library in Pure Python'
```

Comment:
You need a license_file entry. If the license is not in the upstream package, please make an issue there and then include it in this PR per the instructions here: https://conda-forge.org/docs/meta.html#packaging-the-license-manually